test/testdrive/tpch.td

```diff
@@ -52,20 +52,1447 @@ true
+      ps_partkey,
+      sum(ps_supplycost * ps_availqty) AS value
+  FROM
+      partsupp,
+      supplier,
+      nation
+  WHERE
+      ps_suppkey = s_suppkey
+      AND s_nationkey = n_nationkey
+      AND n_name = 'GERMANY'
+  GROUP BY
+      ps_partkey having
+          sum(ps_supplycost * ps_availqty) > (
+              SELECT
+                  sum(ps_supplycost * ps_availqty) * 0.0001
+              FROM
+                  partsupp,
+                  supplier,
+                  nation
+              WHERE
+                  ps_suppkey = s_suppkey
+                  AND s_nationkey = n_nationkey
+                  AND n_name = 'GERMANY'
+          )
+  ORDER BY
+      value DESC;
+103 values hashing to 2a4b6afc2de49adfe1cf81446d355cb1
+
+# Query 12
+> SELECT
+      l_shipmode,
+      sum(case
+          when o_orderpriority = '1-URGENT'
+              or o_orderpriority = '2-HIGH'
+              then 1
+          else 0
+      end) AS high_line_count,
+      sum(case
+          when o_orderpriority <> '1-URGENT'
+              AND o_orderpriority <> '2-HIGH'
+              then 1
+          else 0
+      end) AS low_line_count
+  FROM
+      orders,
+      lineitem
+  WHERE
+      o_orderkey = l_orderkey
+      AND l_shipmode IN ('MAIL', 'SHIP')
+      AND l_commitdate < l_receiptdate
+      AND l_shipdate < l_commitdate
+      AND l_receiptdate >= DATE '1994-01-01'
+      AND l_receiptdate < DATE '1994-01-01' + INTERVAL '1' year
+  GROUP BY
+      l_shipmode
+  ORDER BY
+      l_shipmode;
+2 values hashing to 073eefcd66f3621883f99713bfcca326
+
+# Query 13
+> SELECT
+      c_count,
+      count(*) AS custdist
+  FROM
+      (
+          SELECT
+              c_custkey,
+              count(o_orderkey) c_count
+          FROM
+              customer LEFT OUTER JOIN orders ON
+                  c_custkey = o_custkey
+                  AND o_comment NOT LIKE '%special%requests%'
+          GROUP BY
+              c_custkey
+      ) AS c_orders
+  GROUP BY
+      c_count
+  ORDER BY
+      custdist DESC,
+      c_count DESC;
+27 values hashing to 4db7211f1bec42092d8f3dfea7f385e2
+
+# Query 14
+> SELECT
+      100.00 * sum(case
+          when p_type like 'PROMO%'
+              then l_extendedprice * (1 - l_discount)
+          else 0
+      end) / sum(l_extendedprice * (1 - l_discount)) AS promo_revenue
+  FROM
+      lineitem,
+      part
+  WHERE
+      l_partkey = p_partkey
+      AND l_shipdate >= DATE '1995-09-01'
+      AND l_shipdate < DATE '1995-09-01' + INTERVAL '1' month;
+1 values hashing to f14bf0e7e0cf534cca34f52d32f34ba3
+
+# Query 15
+> SELECT
+      s_suppkey,
+      s_name,
+      s_address,
+      s_phone,
+      total_revenue
+  FROM
+      supplier,
+      revenue
+  WHERE
+      s_suppkey = supplier_no
+      AND total_revenue = (
+          SELECT
+              max(total_revenue)
+          FROM
+              revenue
+      )
+  ORDER BY
+      s_suppkey;
+1 values hashing to e38533aeeb7f6fdfd3c4c95247ef052a
```

Comment:
For things that return < ~10 rows, is it better to print the rows? (I don't care, just asking.)

Reply:
Nah this is fine, just wondering.